doc/abrt-action-install-debuginfo.txt

```diff
@@ -1,13 +1,16 @@
 abrt-action-install-debuginfo(1)
-==============================
+================================
 
 NAME
 ----
 abrt-action-install-debuginfo - Install debuginfos for build-ids
 
 SYNOPSIS
 --------
-'abrt-action-install-debuginfo' [-vy] [--ids=BUILD_IDS_FILE] [--tmpdir=TMPDIR] [--cache=CACHEDIR[:DEBUGINFODIR1:DEBUGINFODIR2...]] [--size_mb=SIZE] [-e PATH]
+'abrt-action-install-debuginfo' [-vy] [--ids=BUILD_IDS_FILE]
+   [--tmpdir=TMPDIR] [--cache=CACHEDIR[:DEBUGINFODIR1:DEBUGINFODIR2...]]
+   [--size_mb=SIZE] [--pkgmgr=(yum|dnf)] [-e PATH[:PATH...]]
+   [--releasever=RELEASEVER] [--repo=PATTERN]
```

Comment:
Seems that the man-page DocBook stylesheet does not spit out newlines here, but I don’t see an easy way out.